doc/md/versioned-migrations.md

```diff
@@ -241,3 +241,115 @@ func main() {
    }
 }
 ```
+
+## Atlas migration directory integrity file
+
+Suppose you have multiple teams develop a feature in parallel and both of them need a migration. If Team A and Team B do
+not check in with each other, they might end up with a broken migration directory since new files do not raise a merge
+conflict in a version control system like git. To prevent concurrent creation of new migration files and guard against
+accidental changes in the migration history, Atlas has a feature called __Migration Directory Integrity File__, which simply is
+another file in your migration directory called `atlas.sum`. 
```

Comment:
I'd also explain why this is good for this use case, how it prevents these scenarios, and how to troubleshoot such conflicts using this file. Please, let me know if you need help with this